wolox-books/src/app/pipes/filter-array-object-by-key-value.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { BookResponse } from '../interfaces/book.interface';
+
+@Pipe({
+  name: 'filterArrayObjectByKeyValue'
+})
+export class FilterArrayObjectByKeyValuePipe implements PipeTransform {
+
+  transform(arrayObject: BookResponse[] | null, key: string, value: string): BookResponse[] | null {
+    if (!arrayObject || arrayObject.length === 0) return arrayObject;
```

Comment:
Remember that the 0 legth is translated to false 
```suggestion
    if (!arrayObject || !arrayObject.length) return arrayObject;
```